CI/before_script.msvc.sh

```diff
@@ -677,8 +677,8 @@ printf "Bullet 2.89 (${BULLET_DBL_DISPLAY})... "
 		printf -- "Exists. (No version checking) "
 	elif [ -z $SKIP_EXTRACT ]; then
 		rm -rf Bullet
-		eval 7z x -y "${DEPS}/Bullet-2.89-msvc${MSVC_YEAR}-win${BITS}${BULLET_DBL}.7z" $STRIP
-		mv "Bullet-2.89-msvc${MSVC_YEAR}-win${BITS}${BULLET_DBL}" Bullet
+		eval 7z x -y "${DEPS}/Bullet-2.89-msvc${MSVC_REAL_YEAR}-win${BITS}${BULLET_DBL}.7z" $STRIP
+		mv "Bullet-2.89-msvc${MSVC_REAL_YEAR}-win${BITS}${BULLET_DBL}" Bullet
```

Comment:
I believe this is the cause of the appveyor problem, revert this and try again?

Reply:
This harks back to this issue: https://github.com/OpenMW/openmw/pull/2937#issuecomment-651921114

It looks like the dependencies are getting vendored from gitlab now though? And the gitlab repo only has it with 2015 in the filename. So I guess it needs to use the "wrong" year now (i.e. 2015). I'll change it and cross my fingers.